docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-potential-credential-access-via-renamed-com-services-dll.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[prebuilt-rule-0-16-2-potential-credential-access-via-renamed-com-services-dll]]
+=== Potential Credential Access via Renamed COM+ Services DLL
+
+Identifies suspicious renamed COMSVCS.DLL Image Load, which exports the MiniDump function that can be used to dump a process memory. This may indicate an attempt to dump LSASS memory while bypassing command-line based detection in preparation for credential access.
```

Comment:
```suggestion
Identifies suspicious renamed COMSVCS.DLL Image Load, which exports the MiniDump function that can be used to dump a process memory. This may indicate an attempt to dump Local Security Authority Server Service (LSASS) memory while bypassing command-line based detection in preparation for credential access.
```